Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see reviewed (approved):
I did the German part. Other languages need to be checked.

ann0see commented on _translator-files/po/de/Installation-for-Linux.po:
Sounds a bit clumsy

ann0see commented on _translator-files/po/de/Installation-for-Linux.po:
Fixed.

pljones commented:
https://hosted.weblate.org/translate/jamulus/1-index/en/?q=has%3Acheck&offset=3

Weblate thinks that needs translating.  Shouldn't it "know" better?

ann0see commented:
I think there's a way to silence the warning.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/
Translation: Jamulus/Privacy-Statement

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/
Translation: Jamulus/Privacy-Statement

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/
Translation: Jamulus/Privacy-Statement

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/
Translation: Jamulus/Privacy-Statement

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/
Translation: Jamulus/Privacy-Statement

ann0see commented:
I'll merge this now. If anything is wrong, please correct it on weblate.
